packages/compass-indexes/src/components/indexes/indexes.tsx

```diff
@@ -74,16 +80,16 @@ export const Indexes: React.FunctionComponent<IndexesProps> = ({
           localAppRegistry={localAppRegistry}
           isRefreshing={isRefreshing}
           writeStateDescription={description}
-          onRefreshIndexes={() => onRefresh()}
+          onRefreshIndexes={refreshIndexes}
```

Comment:
on refresh should we clear `failed` indexes?

<img width="1460" alt="image" src="https://user-images.githubusercontent.com/1305718/187973831-4f8d0bde-68a8-4400-b9d2-cb286c11c067.png">

Reply:
As a user, I would prefer to keep these raws until I explicitly remove them. But we can discuss it with a team during design sync if you have a strong opinion about it.